browser/brave_features_internal_names.h

```diff
@@ -17,5 +17,6 @@ constexpr char kBraveVPNFeatureInternalName[] = "brave-vpn";
 constexpr char kBraveVPNDnsFeatureInternalName[] = "brave-vpn-dns";
 #endif
 #endif
+constexpr char kChatSummarizerFeatureInternalName[] = "chat-summarizer";
```

Comment:
please be consistent with the feature name. Either "AIChat" or "ChatSummarizer" or something else, but only one thing everywhere.

Reply:
I'm not sure if this is used anywhere. We use string literals in about_flags.cc now? I'll remove this as it's not being used.